examples/Batch/WordCountFSharp/WordCountFSharp.fsproj

```diff
@@ -0,0 +1,111 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>17b63d32-efc8-4ef5-831a-197a4fc29f06</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>WordCountFSharp</RootNamespace>
+    <AssemblyName>WordCountFSharp</AssemblyName>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFSharpCoreVersion>4.4.0.0</TargetFSharpCoreVersion>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <Name>WordCountFSharp</Name>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <Tailcalls>false</Tailcalls>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <WarningLevel>3</WarningLevel>
+    <DocumentationFile>bin\Debug\WordCountFSharp.XML</DocumentationFile>
+    <StartArguments>file:///C:/Users/Isaac/Source/Repos/Mobius/build/runtime/data/words.txt</StartArguments>
```

Comment:
I'm not sure what you're referring to here. Configuration Management is not C#-specific, it's a .NET thing. So if it works in C# it should work in F#. The only user settings I'm aware of are related to custom Application Settings / Configuration. Can you give me an example of this .user file?